dpbench/config/benchmark.py

```diff
@@ -0,0 +1,77 @@
+from dataclasses import dataclass
+from typing import Any, List
+
+Parameter = dict[str, Any]
+
+
+@dataclass
+class Parameters:
+    S: Parameter
+    M: Parameter
+    L: Parameter
+
+    @staticmethod
+    def from_dict(obj: Any) -> "Parameters":
+        _S = Parameter(obj.get("S"))
```

Comment:
We should make it more flexible. I can think of scenarios where we have a benchmark info JSON that has different presets based on architectures, _e.g.,_ "S_CPU", "S_GPU", _etc._

Perhaps use a `classmethod` and dynamically populate the attributes from the `keys` of the `dict`?